scripts/dockerfiles/polkadot/docker-compose-local.yml

```diff
@@ -1,35 +1,39 @@
 version: '3'
 services:
   node_alice:
-    build:
-      context: .
-      dockerfile: polkadot_builder.Dockerfile
     ports:
       - "30333:30333"
       - "9933:9933"
       - "9944:9944"
-    image: chevdor/polkadot:latest
+      - "9615:9615"
+    image: parity/polkadot:latest
     volumes:
       - "polkadot-data-alice:/data"
-    command: polkadot --chain=polkadot-local --alice -d /data --node-key 0000000000000000000000000000000000000000000000000000000000000001
+    command: |
+      --chain=polkadot-local
+      --alice
+      -d /data
+      --node-key 0000000000000000000000000000000000000000000000000000000000000001
```

Comment:
does it assume the binary?

Reply:
there is an [entrypoint](https://github.com/paritytech/polkadot/blob/be7dbb2c036d0ba6a979ac22f6b4b3f352ad9fb3/scripts/dockerfiles/polkadot/polkadot_builder.Dockerfile#L36)

Reply:
ah missed the line